admin_solutions/user_role_mgmt.adoc

```diff
@@ -0,0 +1,534 @@
+= User and Role Management
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+As an OpenShift cluster administrator, you are able to manage users, roles, projects, templates, and all available resources. For now, this topic focuses on common operations a cluster administrator needs to perform. You can:
+
```

Comment:
Judging from the title, and the topic headings (though I have yet to go through this in full), I'd suggest a change to the first sentence:

"As an OpenShift cluster administrator, you can manage users and roles, as well as the projects, templates, and available resources." 

Otherwise, it sounds like the reader will use this doc to manage projects and templates and stuff.